### ISSUE TITLE
Better not duplicating repos

### ISSUE THREAD

**Issue by pcav** [AnotherBobSmith/CLUZ_QGIS3#1]
Any special reason to have another plugin for QGIS3, instead of having different versions in the same repository? This second is generally a preferred option, as we try hard not to have too much noise. Most plugins follow this route.
Thanks for considering this opportunity.

**Comment by AnotherBobSmith** [comment]
Hi Paolo,

Thanks for your help with this. I’m happy to have them in the same repository – I just didn’t know that could work. Should I just upload the zip file in the normal way?

Best wishes, Bob

**Comment by pcav** [comment]
Yes, a new version, with the appropriate min and max version. Same GH repo, different releases. See similar plugins for a comparison.
